README.md

```diff
@@ -9,8 +9,11 @@ used at the Biohub and in the broader imaging community.
 ### Read
 
 - OME-Zarr
```

Comment:
consider adding v0.4? And we're not officially supporting v0.1, right? (even though v0.1 works with the legacy reader and shows a warning)